test/parallel/test-events-once.js

```diff
@@ -132,38 +85,24 @@ async function onceError() {
 }
 
 async function onceWithEventTarget() {
-  const et = new EventTargetMock();
-
+  const et = new EventTarget();
+  const event = new Event('myevent');
   process.nextTick(() => {
-    et.dispatchEvent('myevent', 42);
+    et.dispatchEvent(event);
   });
   const [ value ] = await once(et, 'myevent');
-  strictEqual(value, 42);
-  strictEqual(Reflect.has(et.events, 'myevent'), false);
-}
-
-async function onceWithEventTargetTwoArgs() {
```

Comment:
cc @JeniaBR 

I think Domenic was right on https://github.com/nodejs/node/pull/29498#discussion_r328870358 .

This PR still uses the same array wrapping (for compatibility with `EventEmitter` but no longer supports multiple arguments in event dispatch (as that has always been impossible anyway).